tools/congestion-model/src/workload/balanced.rs

```diff
@@ -74,7 +74,7 @@ impl BalancedProducer {
         let attached_gas = 300 * TGAS;
         let execution_gas = 100 * TGAS;
         let conversion_gas = 5 * TGAS;
-        let num_tx_per_shard_pair = 2;
+        let num_tx_per_shard_pair = 4;
```

Comment:
With only 2 tx per pair, I didn't even get congestion with the fixed formula for execution gas.